src/cliparser.jl

```diff
@@ -43,6 +43,11 @@ function parse_commandline()
         help = "use separate line and generator budgets" 
         action = :store_true 
 
+        "--bus_budget"
+        help = "budget for buses. CANNOT BE USED WITH --line_budget OR --generator_budget"
+        arg_type = Int 
+        default = 0
+
```

Comment:
I added this to the CLI interface as well. If we're going to use a dedicated bus-budget argument, we can probably remove the interdict-buses argument, right?